sdks/java/maven-archetypes/examples/src/main/resources/archetype-resources/src/test/java/DebuggingWordCountTest.java

```diff
@@ -20,6 +20,8 @@
 import com.google.common.io.Files;
 import java.io.File;
 import java.nio.charset.StandardCharsets;
+import ${package}.DebuggingWordCount.WordCountOptions;
```

Comment:
Same answer. Just doing a straight replace so that the code continues to be valid. We do have a test that instantiates and builds the archetype examples, right?